src/vs/base/browser/ui/actionbar/actionbar.ts

```diff
@@ -747,7 +747,7 @@ export class ActionBar extends Disposable implements IActionRunner {
 
 	protected updateFocus(fromRight?: boolean): void {
 		if (typeof this.focusedItem === 'undefined') {
-			this.actionsList.focus();
+			this.actionsList.focus({ preventScroll: true });
```

Comment:
The `{ preventScroll: true }` is necessary to avoid scrolling up when focusing the `<ul class="actions-container">`